rcm_nexus/npm.py

```diff
@@ -0,0 +1,214 @@
+# npm stuff
+from __future__ import print_function
+
+import os
+import sys
+import zipfile
+import tarfile
+import tempfile
+import rcm_nexus.config as config
+import base64
+import shutil
+import subprocess
+from .config import DEFAULT_PASSWORD
+
+from enum import Enum
+from hmac import compare_digest
```

Comment:
I believe this doesn't exist on Python 2.6. Maybe just replace it with normal `==`?

Reply:
The compatibility test did not complain. Some of the features were backported. I will see if this will be an issue when testing.